src/components/Filter.jsx

```diff
@@ -1,44 +1,57 @@
-const Filter = ({priorityFilter, setPriorityFilter, statusFilter, setStatusFilter, categoryFilter, setCategoryFilter}) => {
+import { FORM_OPTIONS } from "../constants/FormOptions"
```

Comment:
Deu erro na sua pipeline e provavelmente é por causa dessa linha aqui. O nome do arquivo está na formatação errada. A letra 'f' do 'FormOptions' deveria ser minúscula. Na sua máquina deve ter funcionado porque o Windows suporta isso.

```suggestion
import { FORM_OPTIONS } from "../constants/formOptions"
```